recipes/ms_dotnet3.rb

```diff
@@ -20,10 +20,10 @@
 
 if platform?('windows')
   include_recipe 'ms_dotnet'
-
-  if win_version.windows_server_2008? || win_version.windows_server_2008_r2? || win_version.windows_7? || win_version.windows_vista?
+  if win_version.windows_server_2012? || win_version.windows_server_2012_r2? || win_version.windows_server_2008? || win_version.windows_server_2008_r2? || win_version.windows_7? || win_version.windows_vista?
     windows_feature 'NetFx3' do
       action :install
+      all true
```

Comment:
The `all` attribute is only supported by the `dism` feature provider on windows version greater than 6.2 (windows 8/server 2012)